src/text-field.jsx

```diff
@@ -414,6 +414,10 @@ const TextField = React.createClass({
     }
   },
 
+  _getFontSize() {
+    return this.props.style && this.props.style.fontSize ? this.props.style.fontSize : 16;
```

Comment:
Would you mind showing a more explicit example?

On Sun, Nov 8, 2015 at 1:36 PM, Olivier Tassinari <notifications@github.com>
wrote:

> In src/text-field.jsx
> <https://github.com/callemall/material-ui/pull/2039#discussion_r44231042>:
>
> > @@ -414,6 +414,10 @@ const TextField = React.createClass({
> >      }
> >    },
> >
> > +  _getFontSize() {
> > +    return this.props.style && this.props.style.fontSize ? this.props.style.fontSize : 16;
>
> Could we add a warning in the dev environment if the fontSize is not a
> number?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/callemall/material-ui/pull/2039/files#r44231042>.
>